etc/eth-contracts/hardhat.config.js

```diff
@@ -1,4 +1,4 @@
-require('@nomiclabs/hardhat-ethers');
+require('@nomiclabs/hardhat-waffle');
```

Comment:
Out of curiosity, why using waffle here, instead of ethers?

Reply:
We need to have `hardhat-waffle` as this is adds additional functionality to `chai` (e.g. `.reverted`).
There's no need for `require("@nomiclabs/hardhat-ethers")` as `@nomiclabs/hardhat-waffle` already does it.